src/pb/GridFuncVector.h

```diff
@@ -38,6 +39,8 @@ class GridFuncVector
     static Timer wait_up_down_tm_;
     static Timer wait_east_west_tm_;
 
+    static Map2Masks* map2masks_;
```

Comment:
Does it have to be static? 
Is there anyway to avoid 'static raw pointer' here?

Reply:
It is shared among all objects of that class. But we won't use it for GPU, if this is your concern.